File: src/main/java/org/example/expert/domain/todo/dto/request/TodoPageRequest.java
package org.example.expert.domain.todo.dto.request;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class TodoPageRequest {
    private int page = 1;
    private int size = 10;
    private String weather;
    private LocalDateTime since;
    private LocalDateTime until;
}

File: src/main/java/org/example/expert/domain/todo/dto/request/TodoPageRequest.kt
package org.example.expert.domain.todo.dto.request

import java.time.LocalDateTime

data class TodoPageRequest (
    val page: Long = 1L,
    val size: Long = 10,
    val weather: String? = null,
    val since: LocalDateTime? = null,
    val until: LocalDateTime? = null
)